### PR TITLE
EWPP-914: Ensure drupal 9 compatibility.

### PR DESCRIPTION
**Pull request by nagyad**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,6 +131,29 @@ Then:
 
 ```bash
 docker-compose exec web composer install
+```
+
+This will also:
+- Symlink the theme in  `./build/modules/custom/oe_media` so that it's available for the test site
```
Comment (yenyasinn) [inline README.md]: It won't symlink the theme. Also path is wrong.

Comment (nagyad) [reply README.md]: Copy-pasta

File: composer.json [hunk 2]
```diff
@@ -6,23 +6,22 @@
...
-        "drupal/core": "^8.7",
+        "php": ">=7.3",
+        "drupal/core": "^8.9 || ^9",
         "easyrdf/easyrdf": "1.0.0 as 0.9.1",
-        "openeuropa/rdf_skos": "~1.0"
+        "openeuropa/rdf_skos": "1.x-dev"
     },
     "require-dev": {
         "composer/installers": "~1.5",
         "drupal/core-composer-scaffold": "^8.8",
```
Comment (yenyasinn) [inline composer.json]:
```suggestion
         "drupal/core-composer-scaffold": "^8.9 || ^9",
```

File: composer.json [hunk 3]
```diff
@@ -6,23 +6,22 @@
...
         "drupal/address": "^1.8",
         "drupal/config_devel": "~1.2",
```
Comment (yenyasinn) [inline composer.json:18]:
```suggestion
        "drupal/config_devel": "~1.8",
```

**Comment by yenyasinn** [comment]
Please add name to the testsuite in the phpunit.xml file, otherwise it doesn't pass validation. Please check https://drone.fpfis.eu/openeuropa/oe_corporate_countries/105/21. 
Also add `cacheResult="false"` there also as in another components https://github.com/openeuropa/oe_media/pull/163/files#diff-e35810879ec42bdd81797b3ccb72f6de28a8b4a0e3bfdba43183e133e866b892R2

**Review by yenyasinn — DISMISSED** [review]

**Review by yenyasinn — DISMISSED** [review]

File: phpunit.xml.dist [hunk 4]
```diff
@@ -9,7 +9,7 @@
     <env name="SIMPLETEST_SPARQL_DB" value="sparql://${drupal.sparql.host}:${drupal.sparql.port}/?module=sparql_entity_storage"/>
   </php>
   <testsuites>
-    <testsuite>
+    <testsuite name="OpenEuropa Corporate Countries">
```
Comment (upchuk) [inline phpunit.xml.dist]: OpenEuropa Corporate Countries test suite

**Review by upchuk — CHANGES_REQUESTED** [review]
Install hook takes a parameter `$is_syncing` so we need to use that to check if the install happens in config sync.

Otherwise, looks ok.

**Review by upchuk — APPROVED** [review]

**Review by brummbar — APPROVED** [review]

**Pull request merged**